rest/blip_sync.go

```diff
@@ -237,7 +245,15 @@ func (bh *blipHandler) handleSubscribeToChanges(rq *blip.Message) error {
 	} else if filter != "" {
 		return base.HTTPErrorf(http.StatusBadRequest, "Unknown filter; try sync_gateway/bychannel")
 	}
-	go bh.sendChanges(rq.Sender, since) // TODO: does this ever end?
+
+	// Start asynchronous changes goroutine
+	go func() {
+		base.StatsExpvars.Add("subChanges_total", 1)
+		base.StatsExpvars.Add("subChanges_active", 1)
+		defer base.StatsExpvars.Add("subChanges_active", -1)
```

Comment:
I think it's more idiomatic to put the `defer` statement at the top of the function, or as close to the top as possible.

Reply:
In this case we don't want to execute the defer unless the previous StatsExpvars.Add was successful, so I think we want to leave this as-is.